.claude/scripts/portfolio-surveyor.test.sh

```
@@ -1404,4 +1404,156 @@ grep -Fq 'Requiring **both** `event: dynamic` and a `dynamic/` path' "${surveyor
+grep -Fq 'structural line' "${constitution}" ||
+  fail "AGENTS.md must require structural-line matching for the ownership literals (#2762)"
+
+# The malformed-placement tolerance must not read as an emission convention (Codex P2 on #2767).
+grep -Fq 'DEFECTS, not the convention' "${constitution}" ||
+  fail "AGENTS.md must mark a below-heading disclosure a defect, not the convention (#2762)"
+
+# `none` is a real third state with its own contract: it means neither literal was found, and it is
+# a synonym for NEITHER party. Asserting only that the enum lists three spellings would pass on a
+# definition that never says what `none` means (CodeRabbit 🟡 Minor on #2767).
+grep -Fq 'neither literal' "${surveyor}" ||
+  fail "surveyor must define `none` as neither literal present (#2762)"
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)